scripts/update-readme.js

```diff
@@ -0,0 +1,68 @@
+const { readdirSync, existsSync, readFileSync } = require("fs");
+const { join } = require("path");
+const markdownMagic = require("markdown-magic");
+const tablemark = require("json-to-markdown-table");
+
+const getPathTo = path => join(__dirname, path);
+
+console.log("updateReadme", "Updating README documentation");
+
+const githubURL = "https://github.com/solidjs-community/solid-primitives/tree/main/packages/";
+const sizeShield = "https://img.shields.io/bundlephobia/minzip/";
+const bundlephobiaURL = "https://bundlephobia.com/package/";
```

Comment:
We should probably use bundle.js.org instead as it is more precise.

Reply:
Do they have size badges?

Reply:
Not sure. I think the maintainer is on our discord.

Reply:
Just checked, no, they don't support size badges.